lib/phlex/rails/sgml.rb

```diff
@@ -105,8 +105,10 @@ def render_in(view_context, &erb)
 			context = { rails_view_context: view_context }
 		end
 
+		fragments = context[:rails_view_context].request.headers.fetch("X-Fragments", "").split(" ").presence
```

Comment:
We can just use the `view_context` local here rather than `context[:rails_view_context]`.

Reply:
It will work even if ViewComponent is the renderer?

Reply:
Oh, good point. I don’t think I looked at enough context. You’re probably right.